components/src/dynamo/sglang/request_handlers/embedding/embedding_handler.py

```
@@ -87,6 +91,12 @@ def _transform_response(self, ret, model_name):
             )
             prompt_tokens += ret_item.get("meta_info", {}).get("prompt_tokens", 0)
 
+        try:
+            observe_embedding_batch_size(model_name, len(embedding_objects))
+            observe_embedding_input_tokens(model_name, prompt_tokens)
+        except Exception:
```

Ruff (0.15.13)

[warning] Do not catch blind exception: `Exception`

(BLE001)